README.rst

```diff
@@ -23,10 +24,10 @@ Contact:
   Nathaniel J. Smith <njs@pobox.com> and Stéfan van der Walt <stefanv@berkeley.edu>
 
 Dependencies:
-  * Python 2.6+, or 3.3+
+  * Python 3.7+
```

Comment:
I think we're close to being able to drop 3.7, if that deadline hasn't passed already?

Reply:
Just more than a month left. I think I'll drop it in the next release if that works for you!